### PR TITLE
Update rpi-vi-first-steps.md

### PR DESCRIPTION
**Pull request by irisb1701**
Included some minor modifications of Arran's rpi-vi-first-steps.md

1. Minor grammatical fix
2. Extended list of hardware requirements
3. Replaced broken links 

Please review link below:

https://raw.githack.com/treehouses/treehouses.github.io/slimNew10Steps2/#!rpi-vi-first-steps.md

### CODE REVIEW

**Comment by ArranHL** [comment]
Awesome stuff!

File: rpi-vi-first-steps.md [hunk 1]
```diff
@@ -19,21 +19,29 @@ This equipment will be deployed in the field with little or no access to the int
 
 ### Pre-Requisites
 #### Hardware requirement:
-1. A Raspberry Pi 4 
-1. A Class 10 microSD card (minimal 16GB, but we recommend 32GB or greater)
-1. A microSD card reader (Check if your computer sd slot or if a usb/microSD card adapter is needed)
-1. 8 GB of harddrive space
-1. 8 GB of memory (RAM)
+1.  A Raspberry Pi 4
+	- Pi 3, Pi 3b+, Pi Zero W are also options
```
Comment (ArranHL) [inline rpi-vi-first-steps.md]: We are encouraging them to go straight for RPi 4 so let's remove the others

Comment (irisb1701) [reply rpi-vi-first-steps.md]: Done!

**Comment by ArranHL** [comment]
@irisb1701 It's looking great :)

**Pull request merged**